pkg/nfd-master/nfd-master.go

```diff
@@ -493,20 +615,21 @@ func (m *nfdMaster) UpdateNodeTopology(c context.Context, r *topologypb.NodeTopo
 	return &topologypb.NodeTopologyResponse{}, nil
 }
 
-func (m *nfdMaster) crLabels(r *pb.SetLabelsRequest) map[string]string {
+func (m *nfdMaster) processNodeFeatureRule(r *pb.SetLabelsRequest) (map[string]string, []corev1.Taint) {
 	if m.nfdController == nil {
-		return nil
+		return nil, nil
 	}
 
 	l := make(map[string]string)
+	var crt []corev1.Taint
```

Comment:
nit: just caught my eye now that the name `crt` looks a bit odd here (I'm not sure `l` is a good name either) 😊 How about `taints` or `t`?

Reply:
let's use `taints` and I will update `l` to `labels`